package-lock.json

```diff
@@ -1275,137 +187,44 @@
 			"resolved": "https://registry.npmjs.org/@preact/signals-core/-/signals-core-1.2.3.tgz",
 			"integrity": "sha512-Kui4p7PMcEQevBgsTO0JBo3gyQ88Q3qzEvsVCuSp11t0JcN4DmGCTJcGRVSCq7Bn7lGxJBO+57jNSzDoDJ+QmA=="
 		},
-		"aes-js": {
-			"version": "3.0.0",
-			"resolved": "https://registry.npmjs.org/aes-js/-/aes-js-3.0.0.tgz",
-			"integrity": "sha512-H7wUZRn8WpTq9jocdxQ2c8x2sKo9ZVmzfRE13GiNJXfp7NcKYEdvl3vspKjXox6RIG2VtaRe4JFvxG4rqp2Zuw=="
-		},
-		"bech32": {
-			"version": "1.1.4",
-			"resolved": "https://registry.npmjs.org/bech32/-/bech32-1.1.4.tgz",
-			"integrity": "sha512-s0IrSOzLlbvX7yp4WBfPITzpAU8sqQcpsmwXDiKwrG4r491vwCO/XpejasRNl0piBMe/DvP4Tz0mIS/X1DPJBQ=="
-		},
-		"bn.js": {
-			"version": "5.2.1",
-			"resolved": "https://registry.npmjs.org/bn.js/-/bn.js-5.2.1.tgz",
-			"integrity": "sha512-eXRvHzWyYPBuB4NBy0cmYQjGitUrtqwbvlzP3G6VFnNRbsZQIxQ10PbKKHt8gZ/HW/D/747aDl+QkDqg3KQLMQ=="
+		"@types/node": {
+			"version": "18.15.13",
+			"resolved": "https://registry.npmjs.org/@types/node/-/node-18.15.13.tgz",
+			"integrity": "sha512-N+0kuo9KgrUQ1Sn/ifDXsvg0TTleP7rIy4zOBGECxAljqvqfqpTfzx0Q1NUedOixRMBfe2Whhb056a42cWs26Q=="
```

Comment:
Unfortunately until Ethers fixes their transitive dependency on `@types/node`, we'll end up grabbing this as a transitive dependency.  To deal with this, we will need to add some magic to `tsconfig.json` to get it to ignore global types from `@types/node`.  This is super annoying to do if you actually want global types for some other dependency, but you can see an example of how to do this here:
https://github.com/Zoltu/preact-es2015-template/blob/master/tsconfig.json#L28-L30